src/system/api/rqidApi.ts

```diff
@@ -90,6 +90,7 @@ export class Rqid {
    * @param rqidDocumentType the first part of the wanted rqid, for example "i", "a", "je"
    * @param lang the language to match against ("en", "es", ...)
    * @param scope defines where it will look:
+   * @param best if true only gets the "best" version, highest prio from world, or failing that, highest prio from compendium packs
```

Comment:
@wake42  I added the "best" param to fromRqidRegex.  If this conflicts with your vision of how this might work, I'm open to better ideas.

Reply:
No this looks good, the description is a little of though? It should still care about the scope I think so it gives the "best" according to the requested "scope".

Reply:
If we request "match" or "all" it will return the highest priority item from the world, or failing that the highest priority item from the compendium packs, right, we might want to say that on the description of "match" and "all".  Why is there a difference between "match" and "all"?

Reply:
For `best` & `match` it will do just that, best in world first and if nothing there, best in compendium. 
For `best` & `all` it would get the highest priority of all matches in both world & compendium I guess.

Reply:
Not sure why I want the second option there.  I either want the "best" or I want all of the items that match the regex regardless of prio, right?

Reply:
@wake42 I think you should double check that this works as intended after my changes.